windows-worker/README.md

```diff
@@ -0,0 +1,23 @@
+# zse-windows-worker
+buildbot-worker on windows 10
+
+## Prerequisites
+- git (you need git bash)
+- Microsoft C++ Build Tools
+- python3
+- NSSM - the Non-Sucking Service Manager (https://nssm.cc/)
+- set a password for your user
+## Warning
+Some problem prevents a tls connection between the buildbot-worker on windows and the reverse proxy (traefik) in front of the buildmaster.
+Because of this, tls is deactivated!
```

Comment:
We should aim to activate it

Reply:
I agree. As mentioned I wasn't able to resolve this issue, yet.